src/App.js

```diff
@@ -4,9 +4,61 @@ import './App.css';
 function App() {
```

Comment:
it's not a good practice to define into a file more than 1 component,
please move component to a separate folder wherever needed